doc/_static/custom_scripts.js

```diff
@@ -0,0 +1,162 @@
+            popover: {
+              title: "Context Menu",
+              description: 'To display the context menu for a pane, right-click anywhere over it. The menu shows actions relevant to the element you are hovering over.',
+              position: "right",
+            },
+          },
+          {
+            element: "#editor-rect",
+            popover: {
+              title: "Editor",
+              description: 'The <a href="editor.html">Editor</a> is the pane where you can create, open and edit files. It contains useful features like autocompletion, real-time analysis and syntax highlighting.',
+              position: "right",
+            },
+          },
+          {
+            element: "#console-rect",
+            popover: {
+              title: "IPython Console",
+              description: 'The <a href="ipythonconsole.html">Console</a> allows you to run your code from the Editor or interactively. You can also use it to control Spyder’s debugger.',
+              position: "left",
+            },
+          },
+          {
+            element: "#help-rect",
+            popover: {
+              title: "Help",
+              description: 'The <a href="help.html">Help</a> pane displays documentation for the objects you are using in the Editor or the IPython Console. To trigger Help, press Ctrl-I (Cmd-I on macOS) with your cursor over an object, or type its name in the object field.',
+              position: "left",
+            },
+          },
+          {
+            element: "#variable-explorer-rect",
+            popover: {
+              title: "Variable Explorer",
+              description: 'The <a href="variableexplorer.html">Variable Explorer</a> allows you to browse and interact with the objects generated running your code. Double-clicking a variable will open a specialized viewer, allowing you to inspect its contents.',
+              position: "left",
+            },
+          },
+          {
+            element: "#plots-rect",
+            popover: {
+              title: "Plots",
+              description: 'The <a href="plots.html">Plots</a> pane shows the figures and images created during your code execution. It allows you to browse, zoom, copy, and save the generated plots.',
+              position: "left",
+            },
+          },
+          {
+            element: "#files-rect",
+            popover: {
+              title: "Files",
+              description: 'The <a href="fileexplorer.html">Files</a> pane lets you browse the contents of the directories on your computer, open them in the Editor, and perform a variety of other file operations.',
+              position: "left",
+            },
+          },
+          {
+            element: "#find-rect",
+            popover: {
+              title: "Find",
+              description: 'The <a href="findinfiles.html">Find</a> pane allows you to view and navigate through all occurrences of your search text in a given directory.',
```

Comment:
```suggestion
              description: 'The <a href="findinfiles.html">Find</a> pane allows you to search text in a given directory, and view and navigate through all found occurrences',
```

I think we need to emphasize this pane is meant to search text. The current wording leaves that to the end, which makes its functionality harder to grasp.